benchmarks/release_benchmarking/README.md

```diff
@@ -0,0 +1,191 @@
+./run launch release_benchmarking --extra_args "--print"
+```
+2. Render the document with the Jinja CLI tool:
+```bash
+pushd benchmarks/release_benchmarking
+jinja2 template/results.md.tmpl template/<release-version>.json --format=json > output/<release-version>.md
+```
+
+### (Optional) Generating a PDF report document
+
+You can convert the report to PDF format as an easy way to share your report as a single file
+with embedded plots.
+
+1. In your copy of `template/release.json`, update the `"format"` string to `"pdf"`.
+2. Follow the instructions above to generate your markdown report with Jinja2.
+3. Use `pandoc` to convert the markdown file to PDF:
+```bash
+pushd output
+pandoc <release-version>.md -o <release-version>.pdf --toc
+```
+
+### (Optional) Submitting Results to HoloHub
+
+The Holoscan SDK team may submit release benchmarking reports to HoloHub git history for general visibility. We use Markdown formatting to make plot diagrams accessible for direct download.
+
+1. Move `<release-version>.md` and accompanying plots to a new `release/<version>` folder.
+2. Update image paths in `<release-version.md>` and verify locally with a markdown renderer such as VS Code.
+3. Commit changes, push to GitHub, and open a Pull Request.
+
+## Cleanup
+Benchmarking changes to application YAML files can be discarded after benchmarks complete.
+```bash
+git checkout applications/*.yaml
+```
+
+## Troubleshooting
+
+__Why am I seeing high end-to-end latency spikes as outliers in my data?__
+
+Latency spikes may occur in display-driven benchmarking if the display goes to sleep. Please configure your
+display settings to prevent the display from going to sleep before running benchmarks.
+
+We have also infrequently observed latency spikes in cases where display drivers and CUDA Toolkit
+versions are not matched.
+
+__Benchmark applications are failing silently without writing log files.__
+
+Silent failures may indicate an issue with the underlying applications undergoing benchmarking.
+Try running the applications directly and verify execution is as expected:
+- `./run launch endoscopy_tool_tracking cpp`
+- `./run launch multiai_ultrasound cpp`
+
+In some cases you may need to clear your HoloHub build or data folders to address errors:
+- `./run clear_cache`
+- `rm -rf ./data`
+
+## Developer References
+
+While this tutorial is tailored to curated configurations of the Endoscopy Tool Tracking and
+Multi-AI Ultrasound HoloHub applications, developers utilize underlying
+Holoscan data frame flow tracking tools to similarly measure and analyze performance in custom
+Holoscan applications.
+
+- Refer to the [Holoscan Flow Benchmarking](../holoscan_flow_benchmarking/) project for general
+Holoscan performance profiling tools for both C++ and Python applications.
+- Refer to the [Holoscan Flow Benchmarking whitepaper](https://developer.download.nvidia.com/holoscan/Holoscan-Flow-Benchmarking.pdf) for a comprehensive overview of pipeline profiling tools.
```

Comment:
```suggestion
- Refer to the [Holoscan Flow Benchmarking whitepaper](https://developer.download.nvidia.com/holoscan/Holoscan-Flow-Benchmarking.pdf) and [tutorial](../holoscan_flow_benchmarking/flow_benchmarking_tutorial.md) for a comprehensive overview of pipeline profiling tools.
```

Reply:
Thanks, I missed that the tutorial `.md` and README were two different files